core/src/main/java/com/nisovin/magicspells/spells/targeted/ParticleCloudSpell.java

```diff
@@ -65,7 +67,9 @@ public ParticleCloudSpell(MagicConfig config, String spellName) {
 		particle = ConfigDataUtil.getParticle(config.getMainConfig(), internalKey + "particle", Particle.POOF);
 
 		blockData = getConfigDataBlockData("material", null);
+		argbColor = ConfigDataUtil.getARGBColor(config.getMainConfig(), internalKey + "argb-color", Color.RED);
 		dustOptions = ConfigDataUtil.getDustOptions(config.getMainConfig(), internalKey + "dust-color", internalKey + "size", new DustOptions(Color.RED, 1));
+		dustTransition = ConfigDataUtil.getDustTransition(config.getMainConfig(), internalKey + "color", internalKey + "to-color", internalKey + "size", new Particle.DustTransition(Color.RED, Color.BLACK, 1));
```

Comment:
If we update the spell for parity with the effect, we should not copy over the mistake the effect has of making these options use the same top-level options, and should be in their own section (i.e. `dust-transition`).